packages/jest-emotion/src/matchers.js

```diff
@@ -23,6 +26,8 @@ function isAsymmetric(obj) {
   return obj && isA('Function', obj.asymmetricMatch)
 }
 
+const mediaRegExp = /\([a-z-]+:\s[a-z0-9.]+\)/
```

Comment:
Would this work with `@media only screen` or something else like that?

Reply:
@mitchellhamilton I've change regexp to make media with screen property pass

Reply:
I didn't just mean screen, it was an example, I meant all the media types(https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Media_types) and I'd prefer that the media types aren't hardcoded.